packages/spanner/system-test/spanner.js

```diff
@@ -2716,7 +2716,7 @@ var spanner = new Spanner(env);
           transaction.run('SELECT * FROM TxnTable', function(err, rows) {
             assert.ifError(err);
 
-            assert.strictEqual(rows.length, 2);
+            assert.strictEqual(rows.length, 1);
```

Comment:
Have you tried running all of the Spanner system tests a couple times, not just the one?